src/pages/lib/graphqlapi/subscribe-data/q/platform/[platform].mdx

```diff
@@ -2,3 +2,455 @@ export const meta = {
   title: `Subscribe to real-time events`,
   description: `Learn more about how to observe to data changes using subscriptions in Amplify.`
 };
+
```

Comment:
@renebrandel This one only has the new content under JS and React Native inline filter for the guide. The rest of the platforms use their existing fragments without any guide language or blending.